PyDSS/ProfileManager/hooks/HDF5.py

```diff
@@ -135,3 +130,56 @@ def update(self):
             result = profileObj.update()
             results[profileaName] = result
         return results
+
+class Profile(BaseProfile):
+
+    DEFAULT_SETTINGS = {
+        "multiplier": 1,
+        "normalize": False,
+        "interpolate": False
+    }
+
+    def __init__(self, sim_instance, dataset, devices, solver, mapping_dict, logger, **kwargs):
+        super(Profile, self).__init__(sim_instance, dataset, devices, solver, mapping_dict, logger, **kwargs)
+        self.valueSettings = {x['object']: {**self.DEFAULT_SETTINGS, **x} for x in mapping_dict}
+
+        self.bufferSize = kwargs["bufferSize"]
```

Comment:
This effectively means that `bufferSize` is a required argument. I suggest either making it an argument or providing a default value. Same with `neglectYear`

```
..., logger, bufferSize=X, neglectYear=Y
```

Reply:
this is interface dependend not require by mongo or sql interfaces